dsm/api_client.go

```diff
@@ -79,6 +79,7 @@ func NewAPIClient(endpoint string, port int, username string, password string, a
 	// FIXME: clunky way of creating api_client session
 	tr := &http.Transport{
 		TLSClientConfig: &tls.Config{InsecureSkipVerify: insecure},
```

Comment:
Why are we skipping SSL verification ?

Reply:
We're not @vishwas-ftx - it's only there in the event we need to test it that way (self-signed certificates, etc...). By default it always validated SSL verification, only if "insecure" it sent to true, then it'll skip the SSL verification.

Reply:
Got it